pkg/apitype/stacks.go

```diff
@@ -26,6 +26,9 @@ type StackSummary struct {
 	// LastUpdate is a Unix timestamp of the stack's last update, as applicable.
 	LastUpdate *int64 `json:"lastUpdate,omitempty"`
 
+	// CurentUpdate provides information about the current stack update in-progress as applicable.
```

Comment:
While touching this code and the related service API, would you mind updating the comment on line 26? What value does the `LastUpdate` field take? Is it the timestamp of the last update's "end time"? Or is it when the last update was "started"? Let's update the comment to make that clear and explicit.

Reply:
it's the start time, so I added to comment, I wonder if end time might be better here?  Although more likely to pass 0 that way...

Reply:
Just adding a comment for now is good. But that's something we should definitely fix, since having the start time here isn't really what you want. ("Last updated 2 hours ago" actually means "was updated seconds ago" if the update just finished.)

Could you file an issue for us changing the meaning of this field? It probably isn't important enough for us to fix for a long, long while. And will require us to touch every place where that field is currently read/used. But ultimately it seems like a design wart in the system.

... as far as "more likely to pass 0" if we were to send the end-time, that's OK. Because soon we'll have the `CurrentUpdate` field to include all the relevant details :)